.github/workflows/velociy-dev-s3-deploy.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and Deploy FineractUi to S3
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: Source branch (default `dev`, `dev`/`master`/`staging`/specific branch or tag)
+        default: 'dev'
+        required: true
+      environment:
+        description: Target environment on AWS (default `dev`, `dev`/`staging`/`prod`/other environnment)
+        default: 'dev'
+        required: true
+
+env:
+  SOURCE_REG: 'ghcr.io/velocitynetworkfoundation'
+  SOURCE_REF: ${{ github.event.inputs.ref }}
+  AWS_REGION: 'us-east-1'
+  STACK_NAME: 'velo'
+  ZONE: 'velocitycareerlabs.io'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ env.SOURCE_REF }}
+    # Set Common Variables
+    - name: Set Common Variables
+      run: |
+       # bash ./eng/environments/common.sh
+    - name: Set S3 Bucket Prefix
+      run: echo 'AWS_S3_BUCKET=${{ env.STACK_NAME }}-fineractui-${{ github.event.inputs.environment }}' >> $GITHUB_ENV
+    - name: Build Code In Container
+      run: |
+        docker build . --tag community:latest
+    - name: Run Container for getting artifact
+      run: | 
+        docker run -d -p 80:80 --name communityui community:latest
+    - name: Copy Code to build folder
+      run: |
+        mkdir build
+        docker cp communityui:/usr/share/nginx/html ./build/
+        docker stop communityui
```

Comment:
I feel like I keep saying the same thing over and over again, so here it is in the most simple form:
**DONT BUILD THE APP INSIDE THE CONTAINER!!!!**

Install whatever it needs for building in the pipeline itself, just like here: https://github.com/velocitycareerlabs/monorepo/blob/ad407fb8c8de3ca0a93885a0a805fe755ac7b34f/.github/workflows/deploy-registrar-to-s3.workflow.yml#L34